components/Themes/src/MDCColorScheme.h

```diff
@@ -54,6 +54,17 @@
  */
 @property(nonatomic, strong, nonnull, readonly) UIColor *secondaryDarkColor;
 
+/**
+ The text color to be typically used as titles and subjects. This will currently be used as the color
+ of the alert controller title.
+ */
+@property(nonatomic, strong, nonnull, readonly) UIColor *primaryTextColor;
+/**
+ The text color to be typically used as the body or message. This will currently be used as the color
+ of the alert controller message.
+ */
+@property(nonatomic, strong, nonnull, readonly) UIColor *secondaryTextColor;
+
```

Comment:
Naming is hard.
To have primaryColor, primaryLightColor and primaryDarkColor be completely unrelated to primaryTextColor may be confusing.
Thoughts on alternative naming?  textColor, lightTextColor, darkTextColor?
Do we want to add backgroundColor?  Would that enable us to support Dark and Light theming through our schemes?
Naming is hard!